db/error_handler.cc

```diff
@@ -332,7 +332,7 @@ Status ErrorHandler::RecoverFromBGError(bool is_manual) {
   // For manual recover, shutdown, and fatal error  cases, set
   // recovery_in_prog_ to false. For automatic background recovery, leave it
   // as is regardless of success or failure as it will be retried
-  if (is_manual || s.IsShutdownInProgress() ||
+  if (is_manual || (s.IsShutdownInProgress() || s.IsColumnFamilyDropped()) ||
```

Comment:
Looks like `s` cannot be `ColumnFamilyDropped` here because `ResumeImpl` cannot return `ColumnFamilyDropped`?